main.tf

```diff
@@ -18,13 +18,13 @@ locals {
     base64decode(module.lacework_cfg_svc_account.private_key)
   ))
 
-  default_project_roles = [
+  default_project_roles = var.skip_iam_grants ? [] : [
```

Comment:
We should add a local `skip_iam_grants` that checks whether an existing Service Account is being used also. It looks as though this method would only work when providing an existing SA.